### ISSUE TITLE
TOPIC/GROUP: Admin user sees blank spaces for groups they aren't members of

### ISSUE THREAD

**Issue by ssin1901** [citizenos/citizenos-fe#1597]
**What is the problem?**
_Tested in Mac OS Safari and Google Chrome_

If User Dave has admin rights for a topic and the topic has groups that Dave is not a member of, Dave sees a blank space (screenshot1) with a remove button. However, the remove button will not work and throws up a forbidden error (see screenshot2)

<img width="1385" alt="Screenshot 2024-09-25 at 13 49 24" src="https://github.com/user-attachments/assets/4ad59229-dca6-4c27-a0b4-0a06c6167e72">

<img width="1382" alt="Screenshot 2024-09-25 at 13 49 38" src="https://github.com/user-attachments/assets/8a0b6bd5-3a86-43f9-88a1-a0cf2c4d25bc">

**Why is this a problem?**
Confuses the user

**Possible solution.**
@BeccaMelhuish @ilmartyrk I think that the user should not see the groups that they are not part of? I'm not sure what the UX is actually supposed to be

**Comment by BeccaMelhuish** [comment]
@ssin1901 Yes, agreed they should only see groups there for which they are admin
